romancal/datamodels/tests/test_datamodels.py

```diff
@@ -10,6 +10,12 @@
 from romancal.datamodels.container import ModelContainer, make_file_with_index
 
 
+@pytest.fixture
+def mock_print(mocker):
+    # Mocking the print function to capture outputs for assertions
+    return mocker.patch("builtins.print")
+
```

Comment:
This seems to be the only place where mocker is used. Why do these unit tests need mocker?

Reply:
We needed mocker for mocking the print statement in the newly added unit test for the `.__exit__()` method. However, since the `.__exit__()` method has been refactored, `mocker` is not needed anymore and has been removed.